news/2024/2024-11-17-kel-lan-tournament-3.md

```diff
@@ -0,0 +1,81 @@
+---
+
+Calling all Circle Clickers to Poland! For the third iteration of KEL Tournament, a major in-person tournament with special guests!
+
+![](/wiki/shared/news/2024-11-17-kel-lan-tournament-3/banner.jpg)
+
+KEL3 is a **2-day LAN** tournament at **Kinguin Esports Lounge in Gdańsk, Poland**, standing as the largest event of its kind in the country, where top-tier competition meets an incredible community experience
+
+Depending on amount of accepted invites by special guests, only certain amount of people will qualify to the tournament.
+***Travel and Accommodation expenses are the responsibility of participants and are not covered by the hosts.***
+
+[Signups](https://forms.gle/Kgzww5LPQuwiJMBTA) are open and will run until the **2nd of December**.
+
+Qualifiers will take place online between **4th–18th December**.
+
+**If you signed up for the tournament, you need to join the [Discord](https://discord.com/invite/TSgMXsaZRR)!**
+
+This 2-day event during **February 8th-9th** features a major tournament alongside bundle of side events, all streamed from the [KEL3](https://www.twitch.tv/kinguin_lounge) Twitch channel!
+
+## Schedule
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| **Registrations** | October 8th - December 2nd 2024 |
+| **Qualifiers** | December 4th - December 18th 2024 |
+| **Day 1** | February 8th 2025 (10:00 - 18:00) |
+| **Day 2** | February 9th 2025 (10:00 - 18:00) |
+
+## Prizes
+
+The total prize pool of **610 PLN** will be awarded to the **top 3 players**, along with exciting physical prizes, including the **Wooting UwU Keypads**!
+
+| Placement | Prizes |
+| :-: | :-- |
+| 1st | **Wooting UwU RGB Stealth Keypad** (courtesy of Wooting), 48% of the prizepool,  profile badge (pending) |
+| 2nd | **Wooting UwU Keypad** (courtesy of Wooting), 32% of the prizepool |
+| 3rd | 16% of the prizepool |
+
+## 1v1 double-elimination tournament
+
+The stage is set for an exciting showdown where every match counts. In this fun and fierce 1v1 double-elimination tournament, players will face off in a battle of skill, with every click adding to the excitement and bringing them one step closer to victory!
+
+Below is the bracket schedule:
+
+| Day | Brackets |
+| --: | :-- |
+| **Day 1** | Round of 32, Round of 16, Quarterfinals |
+| **Day 2** | Semifinals, Finals, Grand Finals |
+
+## Side events
+
+During the breaks between or after tournament matches, we'll be hosting a variety of fun side events. Stay tuned for more details coming your way soon!
+
+## Special guests
+
+Wondering who'll be joining the action? Take a look at our first guest reveal:
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/videonotdoneyet" frameborder="0" allowfullscreen></iframe>
+</div>
+
+And that's just the beginning—more special guests will be announced soon! Stay tuned on our [**Discord**](https://discord.com/invite/TSgMXsaZRR) for all the latest updates!
```

Comment:
id nuke this and move it under the participants heading because you talk about them twice currently
```suggestion
```